prdoc/pr_3800.prdoc

```diff
@@ -0,0 +1,12 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Remove deprecated calls from treasury pallet
+
+doc:
+  - audience: Runtime Dev
```

Comment:
Please also target `Runtime User` and list the calls and their replacements.

Reply:
The prdoc is updated. Please help me to check if it is correct. Thank you. https://github.com/paritytech/polkadot-sdk/pull/3820/commits/363270159414a78fd52a24fc8083abad3bc9371d